### PR TITLE
Fix Organizers image hover issue.

### PR DESCRIPTION
**Pull request by jaimemendozadev**
Hey @kpearson, here are my latest changes to the site. Hope it helps!

Deleted img:hover code in _sass/fullstack.scss.
Added 4/6 core values to mission statement with draft copy of value definitions.
Created list of resources page.
Moved Girl Develop It citation to bottom of page.
Changed Machiko Organizer personal webpage icon to Globe in _config.yml.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ people:
       url: https://twitter.com/machikoyasuda
     - title: github
       url: https://github.com/machikoyasuda
-    - title: flash
```
Comment (kpearson) [inline _config.yml:28]: @machikoyasuda which would you like? A :zap: or a 🌎 : ?

Comment (machikoyasuda) [reply _config.yml:28]: Both are great! ⚡ is our first Emoji <3 so i say :zap:

File: _includes/js.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <!-- Plugin JavaScript -->
     <script src="{{ "/js/jquery.easing.min.js" | prepend: site.baseurl }}"></script>
     <script src="{{ "/js/classie.js" | prepend: site.baseurl }}"></script>
+
```
Comment (kpearson) [inline _includes/js.html:10]: @jaimemendozadev what happened here? We can keep it if want this blank line.

Comment (kpearson) [reply _includes/js.html:10]: @jaimemendozadev also so I know going forward, would you like me to modify your commits or comment here and leave it with you make changes?

**Comment by machikoyasuda** [comment]
Love the details PR notes @jaimemendozadev :)

**Comment by jaimemendozadev** [comment]
Oh wow @kpearson . Remember the hovering issue with the organizer photos? I kept fiddling with the JS file by commenting each script on/off and moving them around because I thought it was a JavaScript issue.

Turns out I just had to comment out the offending CSS styling in the scss sheetsheet. My apologies for changing the JS file with the spacing.

By all means change my commits if you think it's better! I just want to make sure I do this project correctly! 😬 

And thanks for the kind words @machikoyasuda! 😊

**Pull request merged**